app/assets/stylesheets/common/components/bookmark-menu.scss

```diff
@@ -0,0 +1,113 @@
+@keyframes slide {
+  0% {
+  }
+  100% {
+    transform: translateY(100%) translateX(-50%);
+    opacity: 0;
+  }
+}
+
+.bookmark-menu-content {
+  &__notice {
+    position: absolute;
+    top: -1.5rem;
+    left: 50%;
+    transform: translateX(-50%);
+    color: var(--tertiary);
+    padding: 0.25rem 0.5rem;
+    white-space: nowrap;
+    opacity: 1;
+
```

Comment:
probably nice to add: "will-change: auto;"

Reply:
Removed this anyway